scripts/fast-reboot

```diff
@@ -663,6 +663,17 @@ if [[ "$REBOOT_TYPE" = "fast-reboot" ]]; then
     fi
 fi
 
+TEAMD_INCREASE_RETRY_COUNT=0
+if [[ "$REBOOT_TYPE" = "warm-reboot" || "$REBOOT_TYPE" = "fastfast-reboot" ]]; then
+    /usr/local/bin/teamd_increase_retry_count.py --probe-only || TEAMD_RETRY_COUNT_PROBE_RC=$?
+    if [[ TEAMD_RETRY_COUNT_PROBE_RC -ne 0 ]]; then
```

Comment:
Are you missing a `$` in variable usage? I think it should be `if [[ $TEAMD_RETRY_COUNT_PROBE_RC -ne 0 ]]; then`

Reply:
That was my initial thinking as well, but then I saw the use of this syntax in other parts of this file (above and below this section), and it worked, so I followed that syntax to match that style. I don't entirely know why it works (is there some feature in bash that checks to see that the string is a variable if it's not a number?), but I think it is better to have it with the `$`.

I'll change it after verifying it doesn't break anything.

Reply:
I think you also need to set it to 0 before line 668? I agree it is better with $.